docs/DevelopmentTips.md

```diff
@@ -45,7 +45,7 @@ DYLD_LIBRARY_PATH=/Library/Developer/Toolchains/swift-LOCAL-YYYY-MM-DD.xctoolcha
 ### Working with two build directories
 For developing and debugging you are probably building a debug configuration of swift. But it's often beneficial to also build a release-assert configuration in parallel (`utils/build-script -R`).
 
-The standard library takes very long to build with a debug compiler. It's much faster to build everything (including the standard library) with a release compiler and only the swift-frontend (with `ninja swift-frontend`) in debug. Then copy the release-built standard library to the debug build:
+The standard library takes very long to build with a debug compiler. It's much faster to build everything (including the standard library) with a release compiler and only the swift-frontend (with `ninja bin/swift-frontend`) in debug. Then copy the release-built standard library to the debug build:
```

Comment:
This looks like a tip for people working on the standard library. Ought this not imply that you want a *debug* standard library built with a release compiler?

Reply:
I'm not sure I understood your question. It doesn't matter if you want to build the stdlib in debug or release. The point is that it's slow to build a stdlib (debug or release) with a debug _compiler_.

Reply:
I get it now, thanks. This may sound like an obvious remark, but maybe it’s worth mentioning that copying over the build artifacts is a separate matter that is meaningful only if you want to *use* the built stdlib with the debug compiler?

Reply:
exactly